src/Illuminate/Queue/RedisQueue.php

```diff
@@ -205,7 +204,7 @@ public function migrateExpiredJobs($from, $to)
      * @param  int  $time
      * @return array
      */
-    protected function getExpiredJobs($transaction, $from, $time)
+    protected function getExpiredJobs(\Predis\Transaction\MultiExec $transaction, $from, $time)
```

Comment:
Adding a typehint to a public or protected method is breaking. Please send this and the related changes to the 5.3 branch, and also can you impprt this at the top please?